notes/WORKSPACES_CMDS.md

```diff
@@ -56,9 +60,19 @@ Same as `cd workspace-a && npm publish` ? This one in particular feel like there
 
 Sets Arb.path to point to the workspace path. More tweaks?
 
+List deps of workspaces the same way we list direct deps of the project today.
+
 ### npm run-script|restart|start|stop|test
 
-Run the script in the given workspace `path`. Should not fail if script missing?
+Run the script in the given workspace `path`.
+
+- Should not fail if script missing?
+  e.g:
+    npm ws run build
+    - if no build script anywhere FAILS
+    - if theres one or more build script: SUCCEED, ignore missing
+- Should it just append `./node_modules/.bin` to `$PATH`?
+- Should it cwd to workspace folder?
```

Comment:
it probably should

Reply:
yeah lots of these questions already have answers 😅  we'll def chat about all this tomorrow in the OpenRFC call